content/en/containers/docker/tag.md

```diff
@@ -22,7 +22,8 @@ If you are running the Agent as a binary on a host, configure your tag extractio
 
 ### Out-of-the-box tagging
 
-The Agent can Autodiscover and attach tags to all data emitted by containers. The list of tags attached depends on the Agent [cardinality configuration][1].
+The Agent can Autodiscover and attach tags to all data emitted by containers. The list of tags attached depends on the Agent [cardinality configuration][1]. [Tag Cardinality][5] can impact billing as different cardinality settings impact the number of emitted metrics.
```

Comment:
is Autodiscover a product or brand name? If it's just functionality, it shouldn't be capitalized. Looks like it's just an adjective.

Reply:
Yes, accepting your suggestion 👍